src/components/Header/MobileHeader/ModileMenu.js

```diff
@@ -9,26 +9,13 @@ import {
 } from "@material-ui/core";
 import MenuIcon from '@material-ui/icons/Menu';
 import ShoppingCartOutlinedIcon from '@material-ui/icons/ShoppingCartOutlined';
-import {withStyles} from "@material-ui/core/styles";
+import {Link as RouterLink} from "react-router-dom";
```

Comment:
`import { Link as RouterLink } from "react-router-dom";`

Reply:
It has conflicts with `Link` from Material UI